### ISSUE TITLE
constraint error on display

### ISSUE THREAD

**Issue by jrmarino** [jrmarino/synth#50]
```
21:44 dillon: marino: synth was unhappy raised CONSTRAINT_ERROR : display.adb:167 range check failed
21:44 dillon: marino: I looked at the source and figured it didn't like the size of the xterm, which is what the problem was. but the error message was pretty opaque
21:51 dillon: marino: I'm running 32/8 but it's got 25-40% idle. will let it go a while longer but maybe reconfigure it for more concurrency.
```

**Comment by jrmarino** [comment]
seems to be caused by a more builders than there are rows

**Comment by jrmarino** [comment]
dillon suggests making the variable part (the log) be at least 10 lines long and let curses overflow so it can be resized bigger later.  curses shouldn't care.

**Comment by jrmarino** [comment]
The listed problem has been fixed but the resizing issue of #24 still exists.
